idl/Microsoft.UI.Xaml.idl

```diff
@@ -136,7 +183,7 @@ namespace MU_X_XTI_NAMESPACE
 
 namespace MU_XC_NAMESPACE
 {
-    [MUX_PREVIEW]
+    [MUX_PUBLIC_V2]
```

Comment:
Should this be MUX_PUBLIC or PUBLIC_V2?

Reply:
So it doesn't matter, but I was illustrating that you can put the attribute wherever. In this case i think it makes sense for the enum to have the same version as the properties that were added.